DevelopedCalculator.js

```diff
@@ -22,7 +22,6 @@ class DevelopedCalculator extends Calculator {
         // pushing the currentResult to the history array 
         this.history.push(this.currentResult);
```

Comment:
Why this line is not needed anymore?

Reply:
Yes I thought we do not need it anymore also
Because we are pushing the current result in operator button

Also We will push it in the equal button later

Thanks